### PR TITLE
DOC-99: Unit tests

### PR DESCRIPTION
**Pull request by yatsenko-ihor**
DOC-106: Write a negative test with one key pair
DOC-107: Write a negative test with two key pair

### CODE REVIEW

File: datasafe-business-impl/src/test/java/de/adorsys/datasafe/business/impl/cmsencryption/CmsEncryptionServiceImplTest.java [hunk 1]
```diff
@@ -31,38 +39,114 @@
...
+
+        OutputStream encryptionStream = cmsEncryptionService.buildEncryptionOutputStream(outputStream,
+                publicKeyIDWithPublicKey.getPublicKey(), publicKeyIDWithPublicKey.getKeyID());
+
+        encryptionStream.write(TEST_MESSAGE_CONTENT.getBytes());
+        IOUtils.closeQuietly(encryptionStream);
+        byte[] byteArray = outputStream.toByteArray();
+
+        ByteArrayInputStream inputStream = new ByteArrayInputStream(byteArray);
+        Assertions.assertThrows(DecryptionException.class, () -> cmsEncryptionService.buildDecryptionInputStream(inputStream, keyStoreAccess));
```
Comment (valb3r) [inline datasafe-business-impl/src/test/java/de/adorsys/datasafe/business/impl/cmsencryption/CmsEncryptionServiceImplTest.java]: Maybe we can use static import?

File: datasafe-business-impl/src/test/java/de/adorsys/datasafe/business/impl/cmsencryption/CmsEncryptionServiceImplTest.java [hunk 2]
```diff
@@ -31,38 +39,114 @@
...
+    @SneakyThrows
+    public void cmsStreamEnvelopeZeroKeyPairFailTest() {
+        PublicKeyIDWithPublicKey publicKeyIDWithPublicKey = keyStoreService.getPublicKeys(keyStoreAccess).get(0);
+        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+
+        CMSEnvelopedDataStreamGenerator gen = new CMSEnvelopedDataStreamGenerator();
+        gen.open(outputStream, new JceCMSContentEncryptorBuilder(CMSAlgorithm.AES128_CBC)
+                .setProvider(BouncyCastleProvider.PROVIDER_NAME).build());
+
+        OutputStream encryptionStream = cmsEncryptionService.buildEncryptionOutputStream(outputStream,
```
Comment (valb3r) [inline datasafe-business-impl/src/test/java/de/adorsys/datasafe/business/impl/cmsencryption/CmsEncryptionServiceImplTest.java]: maybe try with resources may shorten this a bit

File: datasafe-business-impl/src/test/java/de/adorsys/datasafe/business/impl/cmsencryption/CmsEncryptionServiceImplTest.java [hunk 3]
```diff
@@ -31,38 +39,114 @@
...
+        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+        OutputStream encryptionStream = cmsEncryptionService.buildEncryptionOutputStream(outputStream,
+                publicKeyIDWithPublicKey.getPublicKey(), publicKeyIDWithPublicKey.getKeyID());
+        encryptionStream.write(TEST_MESSAGE_CONTENT.getBytes());
+        encryptionStream.close();
+        byte[] byteArray = outputStream.toByteArray();
+
+        KeyStoreAccess newKeyStoreAccess = getKeyStoreAccess(); // new store access would be different from first even was generated similar
+        ByteArrayInputStream inputStream = new ByteArrayInputStream(byteArray);
+        Assertions.assertThrows(CMSException.class, () -> cmsEncryptionService.buildDecryptionInputStream(inputStream, newKeyStoreAccess));
```
Comment (valb3r) [inline datasafe-business-impl/src/test/java/de/adorsys/datasafe/business/impl/cmsencryption/CmsEncryptionServiceImplTest.java]: Maybe we can use static import?

Comment (pn-iyatsenko) [reply datasafe-business-impl/src/test/java/de/adorsys/datasafe/business/impl/cmsencryption/CmsEncryptionServiceImplTest.java]: ok,will fix

File: datasafe-business-impl/src/test/java/de/adorsys/datasafe/business/impl/cmsencryption/CmsEncryptionServiceImplTest.java (hunk 2 above)
Comment (pn-iyatsenko) [reply datasafe-business-impl/src/test/java/de/adorsys/datasafe/business/impl/cmsencryption/CmsEncryptionServiceImplTest.java]: ok, will fix

File: datasafe-business-impl/src/test/java/de/adorsys/datasafe/business/impl/cmsencryption/CmsEncryptionServiceImplTest.java (hunk 1 above)
Comment (pn-iyatsenko) [reply datasafe-business-impl/src/test/java/de/adorsys/datasafe/business/impl/cmsencryption/CmsEncryptionServiceImplTest.java]: ok, will fix

**Pull request merged**